tracing/providers/opentelemetry/src/main/java/io/helidon/tracing/providers/opentelemetry/OpenTelemetryTracer.java

```diff
@@ -107,6 +112,10 @@ public Tracer register(SpanListener listener) {
         return this;
     }
 
+    List<SpanListener> spanListeners() {
+        return Collections.unmodifiableList(spanListeners);
```

Comment:
`List.copyOf(spanListeners)` can be used instead of this.

Reply:
I think the original is what we want.

It is possible for the developer to continue to add listeners after the tracer is created. (Unlikely, perhaps, but possible.) `List.of()` will exclude listeners added later because it makes a copy of the list _at that moment_; as written will include new ones.

Reply:
It is not public API, so probably not important. If public, then we must make sure the docs are aligned with the method chosen.